### ISSUE TITLE
Node Sprint 2 - update timings and stretch coursework

### ISSUE THREAD

**Issue by kfklein15** [CodeYourFuture/PD-curriculum#100]
**What type of request is it**
- [ ] New content
- [x] Update/additional content

**Is the change on:
- [x] Prep
- [x] Day-plan
- [x] Coursework

**Why do you want to suggest this update?**
Feedback from NW6 is that the PD content has time estimations that they cannot fulfil. They also mentioned that most of our coursework is set as mandatory, which it shouldn't.

**Timing process suggestion**
- [ ] Open the current content (intro, prep, exercise, coursework)
- [ ] Paste it into your AI of preference, asking how long someone with a B2 level of English would take to read this
- [ ] Use the slowest or average + X%
- [ ] Update the timing accordingly
- [ ] Review what other content/exercise must be removed

**Coursework**
Per sprint we should have

- 1 key coursework
- 1 Mandatory coursework
- Rest should be stretch coursework

**Document necessary changes as a comment**
- Changes will be done in the system @LaraHuzjan and @kfklein15 
- Must be changed on Module Coursework, CMS and Test Coursework for PD Milestone

**Comment by alfonsocanuk** [comment]
Hello, this is how a came up with timings.
- I used ChtaGPT who gave me always a time range. I always took the middle of the range.
- For videos, I doubled the actual length of the video assuming it may have to be paused, play at slower speed or/and sometimes rolled back to repeat sections.
- For activities, I asked ChatGPT: "You are being asked to facilitate a group activity among 30 students. Given the activity below, how long it will take to run it?". I took the lowest number of the time average in this case, as it seemed a bit excessive. 

========================
**Prep work:**
========================
Intro: 20 minutes
Difficult Conversations At Work (read article): 55 minutes
How To Have Difficult Conversations At Work (read article): 35 minutes
How To Lead Tough Conversations (watch video): 30 minutes

========================
**Day-plan:**
========================
**Preparing For A Difficult Conversation At Work:** 
-----------------------------------------------------------

Introduction and explanation of the activity: 5 minutes
Discussion of each question:
What is the key problem to be addressed?: 5 minutes
What impact does this have on you/the team/work?: 10 minutes
What responsibility are you going to take for your part?: 10 minutes
What do you want to achieve by the end of the conversation?: 10 minutes
Wrap-up and closing remarks: 5 minutes

**Suggestion**: the facilitator can control the time spent on each question to limit it to 5 minutes each but overall to ensure the discussion does not exceed 30 minutes.
**Total: 30 minutes**

**Preparing For A Potentially Conflicting Situation At Work:**
----------------------------------------------------------------------
Introduction and explanation of the activity: 5 minutes
Pairing up students and assigning scenarios: 5 minutes
Role-playing the conversation as a pair for each scenario: 10 minutes per scenario * 10 scenarios = 100 minutes
Transition time between scenarios: 5 minutes
Wrap-up and debrief: 10 minutes

**Suggestion**: assign students (in pairs) one scenario to role-play. If they have time they can pick more. Then ask them to share outcome with the class. In this case timings would be like:

Introduction and explanation of the activity + pairing: 5 minutes
Role-playing the conversation as a pair for one scenario: 10 minutes
Class discussion: 10 minutes
**Total: 25 minutes**

**Comment by LaraHuzjan** [comment]
Looking good @alfonsocanuk, thanks.
This will ensure they have enough time, but we'll be way over the time limit now. 
I have created a new [roadmap](https://docs.google.com/spreadsheets/d/1bTWa9PTG9p5Q0q5Ux8i5Fxt9YrGKJ_gIKoeKNEurXP4/edit#gid=1074899322) that you can use for reference and the timings are all current. As you can see, the prep is already too long for this module, combined with last weeks coursework. 
We have to cut something out in that case, could you have a look at that, please? 
Mark is already working on CW part of the previous week

**Comment by alfonsocanuk** [comment]
Hi @LaraHuzjan how can you tell is too long? what is the magic number we are trying to hit? is it a combination of CW + Prep time?

**Comment by LaraHuzjan** [comment]
"and validate that we are not spending more than 1 hour in the session and 4 hours of combined coursework and prep work". 
The combination of CW and Prep should be max 240 minutes.

**Comment by alfonsocanuk** [comment]
Hi @LaraHuzjan then we have no option but to just limit the prep work to watching the video. The video is 16 minutes and the intro shouldn't take 20 minutes so I do think this can be done in 30 minutes but if it could be possible to lower the CW a few minutes that would be great. So it will be as follows:

**Prep work:**
Intro: 20 minutes
How To Lead Tough Conversations (watch video): 30 minutes (14 minutes padding)

**Day Plan**
No changes to what I suggested which ends up being 55 minutes



By the way, just curious, where did "and validate that we are not spending more than 1 hour in the session and 4 hours of combined coursework and prep work". I couldn't find it on this ticket or on the roadmap doc.

Thanks for your help @LaraHuzjan

**Comment by esma-g** [comment]
@esma-g, I'm just tagging here to follow the conversation.

**Comment by LaraHuzjan** [comment]
@alfonsocanuk It's written on our content template :)
Mark has cut down some timing on the previous week so we could keep more prep in this week. You'll get at least 30 min back for prep, hope that is ok. 
As prep we can keep the ones that we feel are the most useful or just adapt them to fit the timing. 
Thanks :)

**Comment by alfonsocanuk** [comment]
Hi @LaraHuzjan I am unable to see the updated value from Mark in the Roadmap spreadsheet.
Extra 30 minutes will still not be enough unless I break the rules of following the guidance from ChatGPT but it will definitely make it more comfortable for the content I already proposed to keep. 


So to recap:


**========================
Prep work:**
Intro: 20 minutes
How To Lead Tough Conversations (watch video): 30 minutes

**========================
Day-plan:**

--------------------------------------------
**Preparing For A Difficult Conversation At Work:**
--------------------------------------------
Introduction and explanation of the activity: 5 minutes
Discussion of each question:
What is the key problem to be addressed?: 5 minutes
What impact does this have on you/the team/work?: 5 minutes
What responsibility are you going to take for your part?: 5 minutes
What do you want to achieve by the end of the conversation?: 5 minutes
Wrap-up and closing remarks: 5 minutes

**Suggestion**: the facilitator can control the time spent on each question to limit it to 5 minutes each and ensure the whole discussion does not exceed 30 minutes.
**Total: 30 minutes**

--------------------------------------------
**Preparing For A Potentially Conflicting Situation At Work:**
--------------------------------------------
Introduction and explanation of the activity: 5 minutes
Pairing up students and assigning scenarios: 5 minutes
Role-playing the conversation as a pair for each scenario: 10 minutes per scenario * 10 scenarios = 100 minutes
Transition time between scenarios: 5 minutes
Wrap-up and debrief: 10 minutes

**Suggestion**: assign students (in pairs) one scenario to role-play. Ensure there is at leas one pair for each of the scenarios. If they have time they can pick more. Then ask them to share outcome with the class. Timings will be:

Introduction and explanation of the activity + pairing: 5 minutes
Role-playing the conversation as a pair for one scenario: 10 minutes
Class discussion: 10 minutes
**Total: 25 minutes**

**Comment by alfonsocanuk** [comment]
Is this good enough to be reviewed or do I need to make these updates somewhere else? We basically need to update timings but also instructions in the Prep and Day plan content.

**Comment by kfklein15** [comment]
@LaraHuzjan , I think this question is for you.

**Comment by LaraHuzjan** [comment]
@alfonsocanuk yes this is looking good! I updated Mark's timings so in this Node S2 we now have some extra time. 
Prep as you said:
- Intro
- How To Lead Tough Conversations (watch video): 30 minutes
- How to have difficult conversations at work (I'll add this one as we have enough time for a 15 min article)

Day plan stays as you said. 

We just have to look if some of the coursework can be made into stretch. @kfklein15

**Comment by LaraHuzjan** [comment]
I've updated this on CMS.
@kfklein15 I think cw can all stay mandatory in my opinion.

**Comment by LaraHuzjan** [comment]
Actually no, it's too long. If needing to move something to stretch I suggest [AI impact on Tech ](https://github.com/CodeYourFuture/Module-Servers/issues/138) because it's very long @kfklein15

**Comment by kfklein15** [comment]
Update CW.

**Comment by LaraHuzjan** [comment]
@kfklein15 what have we done with the cw, it's too long :)

**Comment by kfklein15** [comment]
Not sure what you mean, @LaraHuzjan . I just moved the coursework to stretch.

**Comment by LaraHuzjan** [comment]
Thank you!
